x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/diff_components/field_diff.tsx

```diff
@@ -46,21 +46,31 @@ export const FieldGroupDiffComponent = ({
   fieldsGroupName,
 }: FieldDiffComponentProps) => {
   const { fieldDiffs, shouldShowSubtitles } = ruleDiffs;
+
   return (
-    <RuleDiffPanelWrapper fieldName={fieldsGroupName}>
+    <SplitAccordion
+      header={
+        <EuiTitle data-test-subj="ruleUpgradePerFieldDiffLabel" size="xs">
+          <h5>{fieldToDisplayNameMap[fieldsGroupName] ?? startCase(camelCase(fieldsGroupName))}</h5>
+        </EuiTitle>
+      }
+      initialIsOpen={true}
+      data-test-subj="ruleUpgradePerFieldDiff"
+    >
       {fieldDiffs.map(({ currentVersion, targetVersion, fieldName: specificFieldName }, index) => {
-        const shouldShowSeparator = index !== fieldDiffs.length - 1;
+        const isLast = index !== fieldDiffs.length - 1;
```

Comment:
```suggestion
        const isLast = index === fieldDiffs.length - 1;
```
I think it should be `===`, otherwise it shows a separator only after the last item.